### PR TITLE
adding tests

### PR DESCRIPTION
**Pull request by alicehsing**

### CODE REVIEW

File: test/functions.test.js [hunk 1]
```diff
@@ -1,24 +1,192 @@
...
+
+    const expected2 = 70;
+    const actual2 = multiplyBySeven (10);
+
+    expect.equal(actual2, expected2, 'the return should be 70');
+
+    const expected3 = -21;
+    const actual3 = multiplyBySeven (-3);
+
+    expect.equal(actual3, expected3, 'the return should be -21');
```
Comment (BenFrancka) [inline test/functions.test.js:47]: Nice job accounting for negative numbers.  Testing for edge cases is an important part of writing reliable tests.
